parcels/fieldset.py

```diff
@@ -383,7 +320,10 @@ def from_netcdf(cls, filenames, variables, dimensions, indices=None, fieldtype=N
                'linear_invdist_land_tracer', 'cgrid_velocity', 'cgrid_tracer' and 'bgrid_velocity'
         :param gridindexingtype: The type of gridindexing. Either 'nemo' (default) or 'mitgcm' are supported.
                See also the Grid indexing documentation on oceanparcels.org
-        :param field_chunksize: size of the chunks in dask loading
+        :param chunksize: size of the chunks in dask loading. Default is None (no chunking). Can be None or False (no chunking),
+               'auto' (chunking is done in the background, but results in one grid per field individually), a four-value
+               tuple (time_chunksize, z_chunksize, y_chunksize, x_chunksize) that only works on A-Grids, or a dict in the format
+               '{parcels_varname: {netcdf_dimname : (parcels_dimname, chunksize_as_int)}, ...}', where 'parcels_dimname' is one of ('time', 'depth', 'lat', 'lon')
```

Comment:
Nice description - I killed the note on the tuple, cause that is firmly removed now.